### ISSUE TITLE
Exception while searching HDSpace. Error: IndexError('list index out of range',)

### ISSUE THREAD

**Issue by PinaColonic** [SickRage/SickRage#2599]
Getting this error with HDSPACE provider quite frequently.

I have debug enabled and do Submit Errors and it seems to work but I do not know where they are going / who is notified about them - so figured I better open an issue here.   Please let me know if there is any other file I can provide.
```
2016-11-06 14:48:49 SEARCHQUEUE-BACKLOG-318408 :: [HDSpace] :: [d15b700] Exception while searching HDSpace. Error: IndexError('list index out of range',) 
2016-11-06 14:48:34 SEARCHQUEUE-BACKLOG-248682 :: [HDSpace] :: [d15b700] Exception while searching HDSpace. Error: IndexError('list index out of range',) 
2016-11-06 14:46:49 SEARCHQUEUE-BACKLOG-75805 :: [HDSpace] :: [d15b700] Exception while searching HDSpace. Error: IndexError('list index out of range',) 
2016-11-06 14:42:44 SEARCHQUEUE-BACKLOG-269586 :: [HDSpace] :: [d15b700] Exception while searching HDSpace. Error: IndexError('list index out of range',) 
2016-11-06 14:38:39 SEARCHQUEUE-BACKLOG-281593 :: [HDSpace] :: [d15b700] Exception while searching HDSpace. Error: IndexError('list index out of range',) 
2016-11-06 14:37:32 SEARCHQUEUE-BACKLOG-78804 :: [HDSpace] :: [d15b700] Exception while searching HDSpace. Error: IndexError('list index out of range',)
```

**Comment by SickRage2** [comment]
Thanks for the issue report! Before a real human comes by, please make sure your report has all the below criteria checked

- [ ] Include basic information: Branch/Commit, OS, What you did, What happened, What you expected
- [ ] Enable debug logging (be sure to disable after the bug is fixed)
- [ ] Post debug logs, either inline (for smaller logs) or using gist

Please make sure you also read [how to create an issue](https://github.com/SickRage/sickrage-issues/blob/master/README.md) and followed all of the steps. 

The title should describe your issue. Having "SR not working" or "I get this bug" for 100 issues, isn't really helpful. We will close issues if there isn't enough information.

Sometimes the devs may seem like grunts and respond with short answers. This isn't (always) because the dev hates you, but because he's on mobile or busy fixing bugs. If something isn't clear, please let us know, and this bot may get updated to automatically answer you.

Thanks!

**Comment by neoatomic** [comment]
See if you can post your sickrage version information etc. Also when did this start and dit it work fine before.?

Maybe HDSPACE changed something or implemented a Captcha that is coursing this.

**Comment by PinaColonic** [comment]
SickRage Info:
Branch: master
Commit: d15b7003f8663b76d01fcbf0566389b2f31e0564
Version: v2016.10.28-1
Database Version: 44.0

Thank you for your time & response.   I am not sure exactly when it started or if it has been forever - I can say that I have seen this error infrequently for months now but it seems like it is more frequent recently, i.e. I see the errors to clear more often for the last couple weeks which drove me to open this issue report.   I have seen a captcha visiting their website with my browser in the past but am not sure if it is currently active or if it was temporary to help with DDOS or Spam or something.   I am still getting these errors as of Nov 10, in any case.

**Comment by b0mmer** [comment]
Seeing the same issue with HDSpace

SickRage Info:
Branch: master
Commit: a1bf3b
Version: v2016.10.28-1
Database Version: 44.0
Linux-3.16.0-4-amd64-x86_64-with-Ubuntu-14.04-trusty

I've noticed the issue off and on since July. For the past two weeks it has been showing up during every backlog search.

Debugging shows:
`20:13:55` ERROR::SEARCHQUEUE-BACKLOG-75978 :: [HDSpace] :: [a1bf3b3] Exception while searching HDSpace. Error: IndexError('list index out of range',)
Traceback (most recent call last):
  File "/app/sickrage/sickbeard/search.py", line 479, in searchProviders
    searchResults = curProvider.find_search_results(show, episodes, search_mode, manualSearch, downCurQuality)
  File "/app/sickrage/sickrage/providers/GenericProvider.py", line 152, in find_search_results
    items_list += self.search(search_string, ep_obj=episode)
  File "/app/sickrage/sickbeard/providers/hdspace.py", line 115, in search
    data = data.split('<div id="information"></div>')[1]
IndexError: list index out of range`

I manually checked and my last 5 occurrences of the error are on episode searches that HDSpace has no results. When I force an episode search for any item that HDSpace does have there is no issue.
